op_crates/console/lib.deno_console.d.ts

```diff
@@ -0,0 +1,29 @@
+// Copyright 2018-2021 the Deno authors. All rights reserved. MIT license.
```

Comment:
Why call this file lib.deno_console.d.ts instead of lib.console.d.ts?

Reply:
The d.ts files are all named the same as the op crate that contains them: `lib.deno_web.d.ts`, `lib.deno_webgpu.d.ts` etc.